README.rst

```diff
@@ -14,6 +14,19 @@ Kebechet
 Kebechet is an extensible system of repository managers for GitHub, GitLab, and
 Pagure.
 
+
+To configure Kebechet on the repo, visit https://github.com/apps/khebhut
```

Comment:
```suggestion
To configure Kebechet on the repo, visit https://github.com/marketplace/khebhut
```
switch it with this link, it would later get the folks to the same link you had earlier
still it would atleast be more professional to have marketplace line , i think.

Reply:
Got it, Fixed !